lib/src/delegates/asset_picker_text_delegate.dart

```diff
@@ -1134,3 +1138,98 @@ class KoreanAssetPickerTextDelegate extends AssetPickerTextDelegate {
   @override
   String get sUnitAssetCountLabel => '개';
 }
+
+/// [AssetPickerTextDelegate] implements with Traditional Chinese.
+/// Traditional Chinese Localization
+class TraditionalChineseAssetPickerTextDelegate
```

Comment:
Does the delegate match the exact texts in the WeChat with Traditional Chinese? If not please make it aligned text-by-text.

Reply:
Switch the language in the settings. These are available on both iOS and Android.

![Screenshot_2024-12-12-06-46-46-73_e39d2c7de19156b0683cd93e8735f348](https://github.com/user-attachments/assets/919d910b-49fa-40ac-a354-f92f41722a7e)

Reply:
It seems like the asset picker in WeChat doesn't look like the same. But I have made some changes according to the texts in WeChat based on my observation.